vue3/src/components/HelloWorld.vue

```diff
@@ -0,0 +1,13 @@
+<script setup>
+defineProps({
+  msg: String,
+});
+</script>
+
+<template>
+  <h1>{{ msg }}</h1>
```

Comment:
And noticed a few more things - can we replace this with our `gcds-heading` component please?

`<gcds-heading tag="h1">{{ msg }}</gcds-heading>`